jsf/schema_types/string_utils/content_type/text__plain.py

```diff
@@ -3,17 +3,23 @@
 LOREM = """Lorem ipsum dolor sit amet consectetur adipisicing elit.
 Hic molestias, esse veniam placeat officiis nobis architecto modi
 possimus reiciendis accusantium exercitationem quas illum libero odit magnam,
-reprehenderit ipsum, repellendus culpa!""".split()
+reprehenderit ipsum, repellendus culpa! Nullam vehicula ipsum a arcu cursus vitae congue.
+Enim nec dui nunc mattis enim ut tellus.""".split()
 
 
 def random_fixed_length_sentence(_min: int = 0, _max: int = 50) -> str:
+    if _min > _max:
+        raise ValueError("'_max' should be greater than '_min'")  # pragma: no cover
     output = ""
-    while len(output) < _max:
+    while True:
         remaining = _max - len(output)
-        valid_words = list(filter(lambda s: len(s) < remaining, LOREM))
+        valid_words = list(filter(lambda s: len(s) <= remaining, LOREM))
         if len(valid_words) == 0:
             break
-        output += random.choice(valid_words) + " "
-        if len(output) > _min and random.uniform(0, 1) > 0.9:
+        if len(output) >= _min and random.uniform(0, 1) > 0.5:
```

Comment:
Any reason to change from 0.9 to 0.5?

Reply:
Not really. I was trying something manually and left it. I will roll back it.